src/content/platform-overview/architecture/index.md

```diff
@@ -0,0 +1,156 @@
+- CI/CD: Create your pipelines to build and deploy the applications.
+- Registry: Store application artifacts in a secure and available place.
+- Smart Routing: Configure ingress/egress access to your applications with advanced service mesh capabilities.
+- Policy Enforcement: Ensure compliance with company policies across the platform landscape.
+- Resource provisioning: Enable easy provisioning and configuration of external resources.
+
+### Infrastructure
+
+For managing the infrastructure, we run a management cluster per provider and region where you want to have your workloads. From that management cluster, you can spin up as many individual Kubernetes clusters, called workload clusters, as you want. Our operations team works to maintain all cluster components' health, and continuously releases new versions with new features and patches.
+
+Giant Swarm's cluster architecture is split into two logical parts. One encompasses the management cluster and all the components running there. The second part refers to the workload clusters that are created dynamically by the users to run their business workloads. In principle, the management cluster and workload cluster(s) are analogous in terms of infrastructure and configuration. The difference comes with the additional layers we deploy on top of the management cluster that helps manage your users and permissions, workload clusters, and the applications running on the workload clusters.
+
+As explained previously, both the management cluster and the workload cluster(s) have the same structure and configuration. Giant Swarm also collaborates on [Cluster API](https://cluster-api.sigs.k8s.io/) to simplify the bootstrap and configuration of the cluster infrastructure and all the components needed for a cluster to function.
+
+![Cluster architecture image](CAPI_architecture.png)
+
+By default, the machines are spread across three different failure domains, or zones, to ensure the availability of the API and workloads running on top. In our setup, three control plane machines hold the Kubernetes API and the other controllers, and a variable number of worker machines contain the workloads.
+
+Besides the Kubernetes machines, we run a bastion host that helps us with the operations. It is the single entry point to the running infrastructure. That way all cluster machines can live in a private network and reduce the exposure of the services running on them via explicit configuration. We are working on a [Teleport](https://goteleport.com/) solution to replace the necessity of the bastions and centralize cluster access through a single tool.
+
+#### Providers
+
+Although we try to maintain feature parity between infrastructure providers, offering the same features and using the same configuration, there are some subtle differences depending on the underlying provider functionality.
+
+{{< tabs >}}
+{{< tab id="flags-clouddirector" title="VMware Cloud Director">}}
+
+*Compatibility*
```

Comment:
Let's use h5 headings here

Reply:
and maybe CSS rule `.tab-pane > h5:first-child { margin-top: 0 }` so it doesn't look awkward